controllers/touristAttractionController.js

```diff
@@ -8,3 +9,56 @@ exports.getAllAttractions = async (req, res) => {
     res.status(500).json({ message: error.message });
   }
 };
+
+//METODO POST
+exports.createAttractions = async (req, res) => {
+  try {
+    const createAttraction = new TouristAttraction({
+      attractionId: req.body.id,
+      name: req.body.name,
+      description: req.body.description,
+      location: req.body.location,
+      image: req.body.image
+    })
+    const newAttraction = await createAttraction.save();
+    res.status(201).json(newAttraction);
+  } catch (error) {
+    res.status(400).json("No se pudo crear la atraccion.")
+  }
+}
+
+//METODO PUT
+exports.updateAttractions = async (req, res) =>{
+  try {
+    const updateAttraction = await TouristAttraction.findByIdAndUpdate(req.params.id, req.body, {
+      new: true
+    });
+    res.status(201).json(updateAttraction);
+  } catch (error) {
+    res.status(400).json("No se pudo modificar la atraccion.");
+  }
+}
+
+//METODO DELETE
+exports.deleteAttractions = async (req, res) =>{
+  try {
+    const deleteAttraction = await TouristAttraction.findByIdAndDelete(req.params.id);
+    if(!deleteAttraction){
+      res.status(404).json("Atraccion no encontrada");
+    }
+    res.status(201).json("Atraccion borrada correctamente.");
+  } catch (error) {
+    res.status(400).json("No se pudo eliminar la atraccion.")
+  }
+}
+
+
+//METODO GET byID
+exports.getAttractionsById = async (req, res) =>{
+  try {
+    const getAttractionsById = await TouristAttraction.findById(req.params.id);
```

Comment:
`getAttractionsById` is not the right name for this env, it should be something like `attraction`, remember this env will storage the information returned by mongo that match with the id that we send in `findById ` method.